tests/src/test/scala/whisk/core/controller/test/ActivationsApiTests.scala

```diff
@@ -462,6 +462,58 @@ class ActivationsApiTests extends ControllerTestCommon with WhiskActivationsApi
     }
   }
 
+  it should "skip activations and return correct ones" in {
+    implicit val tid = transid()
+    val activations = (1 to 3).map { _ =>
+      WhiskActivation(
+        namespace,
+        aname(),
+        creds.subject,
+        ActivationId.generate(),
+        start = Instant.now,
+        end = Instant.now)
+    }.toList
+
+    try {
+      activations.foreach(storeActivation)
+      waitOnListActivationsInNamespace(namespace, 3)
```

Comment:
The "3" can be derived fromt the activations list: `waitOnListActivationsInNamespace(namespace, activations.size)`